_posts/2020-10-23-kubernetes-webhooks-made-easy-with-openfaas.md

```diff
@@ -0,0 +1,265 @@
+---
+title: "Kubernetes Webhooks made easy with OpenFaaS"
+description: "Extending the Kubernetes with Admission webhook using OpenFaaS Serverless Functions"
```

Comment:
`In this post you'll learn how to write Kubernetes Admission webhooks using OpenFaaS functions`

Reply:
done.